### ISSUE TITLE
P1:  Mounting file system - additional checks and troubleshooting

### ISSUE THREAD

**Issue by nanometrenat** [servalproject/lbard#7]
We have experienced a number of intermittent issues with various Mesh Extenders recently.  It seems that on startup something is causing the file system to not mount as it should.

S99mountcheck needs to be fixed.  Symptoms:  on device 1CAFA650 in Epau (which has Nov 16 firmware version) the splash page etc. were loaded onto the root file system and should not have been.  This also meant it ran out of space.  This was identified on 18 Nov so Paul deleted the splash page to temporarily help the issue.  On 19 Nov we saw that this device was still having problems - rebooting constantly, it also had the SSID issue intermittently (https://github.com/servalproject/openwrt-packages/issues/10)

The Mount Check needs to happen BEFORE ServalD starts.

Need to ensure the mounting is reliable.  Separate issue being logged for the Radio mounting.
